modules/command-engine/core/src/main/scala/surge/internal/kafka/KafkaProducerActorImpl.scala

```diff
@@ -122,6 +147,9 @@ class KafkaProducerActorImpl(
   private implicit val rates: AggregateStateRates = AggregateStateRates(aggregateName, metrics)
 
   context.system.scheduler.scheduleOnce(10.milliseconds, self, InitTransactions)
+
+  context.system.scheduler.scheduleAtFixedRate(10.milliseconds, 10.seconds, self, ClearExpiredTrackers)
```

Comment:
Can you cancel this task on shutdown as well? Similar to `flushMessagesScheduledTask` below

Reply:
done